docs/django-admin/roles.md

```diff
@@ -9,6 +9,28 @@ our `user_group` model and run in a migration.
 
 For more details, refer to the [user group model](../../src/registrar/models/user_group.py).
 
+## Adding a user as analyst or granting full access via django-admin
+
+If a new team member has joined whose data is NOT included in fixtures,
+then they will need to be granted analyst ( `cisa_analysts_group`) or full access (`full_access_group`) permissions in order to view the django-admin pages.
+To do this, do the following:
+
+1. The user in question will need to have a login.gov account and login into our system, this will create a `Users` table entry with their email address and name.
+2. On that `Users` table note that the `GROUP` column should be blank for them as they have no special permissions yet.
+3. Click on their username, then scroll down to the `Permissions` section.
```

Comment:
**nit:** `Permissions` -> `User Permissions` (and would be same for in step 4 too if you decide to change and in step 2 for removing section)

(A total extreme nitpick, and feel free to ignore these if it's not useful --  I'm just following along with the instructions and double checking and dunno if it was important making sure we had the exact section name 😅)